frontend/src/metabase-types/types/Event.ts

```diff
@@ -0,0 +1,24 @@
+import { CollectionId } from "./Collection";
+import { User } from "./User";
+
+export interface Event {
```

Comment:
These types should go to `metabase-types/api`, `metabase/types` is deprecated and invalid

Reply:
Oh, gotcha, that felt a bit confusing. Let me move them. Thank you